arrows/ffmpeg/ffmpeg_video_input.cxx

```diff
@@ -240,111 +264,120 @@ class ffmpeg_video_input::priv
       this->f_backstep_size = 12;
     }
 
-    this->f_video_stream = this->f_format_context->streams[this->f_video_index];
+    this->f_video_stream =
+      this->f_format_context->streams[ this->f_video_index ];
     this->f_frame = av_frame_alloc();
     this->f_filtered_frame = av_frame_alloc();
     this->f_packet = av_packet_alloc();
 
     // The MPEG 2 codec has a latency of 1 frame when encoded in an AVI
     // stream, so the pts of the last packet (stored in pts) is
     // actually the next frame's pts.
-    if (this->f_video_stream->codecpar->codec_id == AV_CODEC_ID_MPEG2VIDEO &&
-      std::string("avi") == this->f_format_context->iformat->name)
+    if( this->f_video_stream->codecpar->codec_id == AV_CODEC_ID_MPEG2VIDEO &&
+        std::string( "avi" ) == this->f_format_context->iformat->name )
     {
       this->f_frame_number_offset = 1;
     }
 
     this->f_start_time = this->f_video_stream->start_time;
     this->frame_advanced = false;
-    this->f_frame->data[0] = NULL;
+    this->f_frame->data[ 0 ] = NULL;
     return true;
   }
 
   // ==================================================================
+
   /*
-  * @brief Close the current video.
-  */
-  void close()
+   * @brief Close the current video.
+   */
+  void
+  close()
   {
     this->f_video_index = -1;
     this->metadata.clear();
     this->f_start_time = -1;
 
-    if (this->f_video_stream)
+    if( this->f_video_stream )
     {
       this->f_video_stream = nullptr;
     }
 
-    av_frame_free(&this->f_frame);
-    av_frame_free(&this->f_filtered_frame);
-    av_packet_free(&this->f_packet);
-    avformat_close_input(&this->f_format_context);
-    avformat_free_context(this->f_format_context);
-    avcodec_free_context(&this->f_video_encoding);
-    avfilter_graph_free(&this->f_filter_graph);
+    av_frame_free( &this->f_frame );
+    av_frame_free( &this->f_filtered_frame );
+    av_packet_free( &this->f_packet );
+    avformat_close_input( &this->f_format_context );
+    avformat_free_context( this->f_format_context );
+    avcodec_free_context( &this->f_video_encoding );
+    avfilter_graph_free( &this->f_filter_graph );
   }
 
   // ==================================================================
+
   /*
-  * @brief Initialize the filter graph
-  */
-  bool init_filters(std::string const& filters_desc)
+   * @brief Initialize the filter graph
+   */
+  bool
+  init_filters( std::string const& filters_desc )
   {
-    auto deleter = [](AVFilterInOut** ptr)
-    {
-      avfilter_inout_free(ptr);
-      delete[] ptr;
-    };
-    using AVFilterInOut_ptr = std::unique_ptr<AVFilterInOut*, decltype(deleter)>;
-    char args[512];
+    auto deleter = [ ]( AVFilterInOut** ptr ) {
```

Comment:
I fixed that one, but if this problem is effecting every lambda, I think it might just be better to fix it after Uncrustify can do this for all lambdas.